AUTHORS

```diff
@@ -119,6 +119,11 @@ listed below by date of first contribution:
 * signalw
 * Laurent Chriqui (laurent-chriqui)
 * Andrii Oriekhov (andriyor)
+* Tom Boshoven (TBoshoven)
+* Jaakko Lappalainen (jkklapp)
+* Oday Alhayek (RioAtHome)
+* Christian Clauss (cclauss)
+* meetshah133
```

Comment:
this may be reverted, since it's here due to a cherry pick

Reply:
`TBoshoven` we might want to leave, if we are to include that `Forwarded` bugfix.

Reply:
re-reverting!